IBIT + Oxímetro + Nuvem (Versao03_Release01)

jhonatantcn opened the pull request:
--> Junção da última versão do IBIT multimodal com o código de Adam Mews [envio de dados para a nuvem];
--> BugFix: O Score final do CakeGame não estava sendo mostrado;
--> Foi criada uma conexão entre Oxímetro e IBIT (Plataforma e Minigames), onde é mostrada a frequência cardíaca e a porcentagem de oxigenação no sangue do paciente durante o jogo.
--> Substituição do meu script "SerialControllerPitaco.cs", pelo script de Adam Mews, pois o meu continha alguns trechos a menos de código.

Jogo: I BLUE IT
Versão: 03
Release: 01

jhonatantcn committed:
Merge pull request #3 from UDESC-LARVA/master

Atualização "Adicionado a API e seus DTOs"

jhonatantcn committed:
Revert "Merge pull request #3 from UDESC-LARVA/master"

This reverts commit 966e1b8db4eb294d93d4b2abe67d71c699244c15, reversing
changes made to fb4ba6fcd8b9ace655c380743cf5bc146c97b82b.

jhonatantcn committed:
IBIT + Oxímetro + Nuvem (Versao03_Release01)

--> Junção da última versão do IBIT multimodal com o código de Adam Mews [envio de dados para a nuvem];
--> BugFix: O Score final do CakeGame não estava sendo mostrado;
--> Foi criada uma conexão entre Oxímetro e IBIT (PLataforma e Minigames), onde é mostrada a frequência cardíaca e a porcentagem de oxigenação no sangue do paciente durante o jogo.

Jogo: I BLUE IT
Versão: 03
Release: 01

jhonatantcn committed:
Merge branch 'master' of https://github.com/jhonatantcn/IBLUEIT-Testes

jhonatantcn committed:
Bug Fix

Exclusão de alguns arquivos .csv da pasta savedata enviados por engano.

jhonatantcn committed:
Bug Fix

Substituição do meu script "SerialControllerPitaco.cs", pelo script de Adam Mews, pois o meu continha alguns trechos a menos de código.

damews commented on Assets/_Game/Scripts/CakeGame/RoundManager.cs:
```diff
@@ -22,46 +22,72 @@ public class RoundManager : MonoBehaviour
-                    //Para voltar os outros controladores, voltar a condição AND com scm e scc
-                    while (!scp.IsConnected)
+                    
+                    while (!scp.IsConnected && !scm.IsConnected && !scc.IsConnected)
```
O jogador conseguirá jogar somente com o PITACO com as condições encadeadas com &&?

jhonatantcn replied:
Neste trecho é verificado se nenhum dispositivo de controle está conectado, se nenhum estiver, o jogo imprime a msg "Nenhum dispositivo de controle conectado! ......." e não deixa o jogo começar.

Sobre os controles nos minigames, os pacientes só conseguirão jogar isoladamente e seguindo uma ordem de precedência:
--> Se conectar Pitaco, Mano e Cinta = Só o Pitaco funcionará;
--> Se conectar Mano e Cinta = Só o Mano funcionará;
--> Se conectar Cinta sozinha = Só assim que a Cinta funcionará.

Ordem de Precedência: 1º Pitaco, 2º Mano, 3º Cinta.

Apenas na Plataforma haverá uso simultâneo de dispositivos.

damews replied:
Ok

damews commented on Assets/_Game/Scripts/Core/Data/Pacient.cs:
```diff
@@ -122,29 +122,29 @@ public static Pacient MapFromDto(PacientDto pacientDto)
-                    InsPeakFlow = pacientDto.CapacitiesPitaco.InsPeakFlow,  //valor original -330
-                    ExpFlowDuration = pacientDto.CapacitiesPitaco.ExpFlowDuration,   //valor original
-                    InsFlowDuration = pacientDto.CapacitiesPitaco.InsFlowDuration   //valor original
+                    RespiratoryRate = pacientDto.CapacitiesPitaco.RawRespiratoryRate,
```
Essa alteração já não foi feita?

jhonatantcn replied:
Pior que já, sei lá porque o GitHub apontou que não...

jhonatantcn committed:
Bug Fix - Comentando o Pitaco Simulado

Comentei os trechos de código contendo o controle Pitaco simulado, em "SerialControllerPitaco" para poder usar o Pitaco físico no jogo.

damews committed:
Merge branch 'master' into master
